finality-aleph/src/network/split.rs

```diff
@@ -77,17 +131,22 @@ async fn forward_or_wait<
     receiver: &Arc<Mutex<R>>,
     left_sender: &mpsc::UnboundedSender<LeftData>,
     right_sender: &mpsc::UnboundedSender<RightData>,
+    name: &String,
 ) -> bool {
     match receiver.lock().await.next().await {
         Some(Split::Left(data)) => {
             if left_sender.unbounded_send(data).is_err() {
-                warn!(target: "aleph-network", "Failed send despite controlling receiver, this shouldn't've happened.");
+                // this is totally fine - the other half of the channel can be dropped by any reason
+                // but it's not our responsibility to track it here
```

Comment:
```suggestion
                // This is totally fine - the other half of the channel can be dropped for any reason
                // but it's not our responsibility to track it here.
```

Reply:
Suggestion/nit: have this comment once before the `match` to reduce visual noise.

Reply:
moved